tests/test_connection.py

```diff
@@ -377,6 +377,61 @@ class MemoryListener(trio.abc.Listener):
     with trio.fail_after(2):
         async with open_websocket(
                 HOST, server.port, '/', use_ssl=False) as connection:
-            with pytest.raises(ConnectionClosed) as e:
+            with pytest.raises(ConnectionClosed) as exc_info:
                 await connection.get_message()
-                assert e.reason.name == 'NORMAL_CLOSURE'
+            exc = exc_info.value
```

Comment:
I should have mentioned this in the commit message. This assertion was not getting executed because the previous line `await connection.get_message()` was raising an (expected) exception. In order to run this assertion, it needs to happen outside the `pytest.raises` context manager.